common/src/main/java/gov/cms/ab2d/common/config/Mapping.java

```diff
@@ -51,7 +51,7 @@ public void init() {
 
         Converter<Contract, ContractDTO> contractContractDTOConverter = context ->
                 new ContractDTO(context.getSource().getContractNumber(), context.getSource().getContractName(),
-                                context.getSource().getAttestedOn().toString());
+                                context.getSource().getAttestedOn().toString(), context.getDestination().getContractType());
```

Comment:
Is getSource() a Contract?  If so, just use the toDTO() method.

If not, what is it?